PWGHF/HFC/DataModel/DerivedDataCorrelationTables.h

```diff
@@ -81,7 +81,17 @@ DECLARE_SOA_TABLE(DsCandSelInfos, "AOD", "DSCANDSELINFO", //! Table with Ds cand
                   aod::hf_candidate_reduced::BdtScorePrompt,
                   aod::hf_candidate_reduced::BdtScoreBkg);
 
-DECLARE_SOA_TABLE(HfcRedCharmHads, "AOD", "HFCREDCHARMHAD", //! Table with charm hadron candidate info
+DECLARE_SOA_TABLE(HfcRedCharmHads2P, "AOD", "HFCREDCHARMHAD2P", //! Table with 2-prong charm hadron candidate info
```

Comment:
The description `HFCREDCHARMHAD2P` has 17 characters which will get truncated and might lead to naming conflicts.